### PR TITLE
Bugfix/cloning environment

### PR DESCRIPTION
**Pull request by gerardcl**

### CODE REVIEW

**Comment by henrjk** [comment]
# About this PR

**TLDR:** The rolebinding named `admin` from the cloned source environment is lost during cloning and instead replaced by the rolebinding created from `oc new-project`. This results nonetheless in a usable cloned environment and fixes the issues listed by Gerard.

**Proposal:** Once this is merged, open an issue to handle this better in 2.0. I believe that using tailor could simplify the cloning scripts a lot. 

**Details:**

`oc new-project <project-name>`  creates a role binding with name `admin`, for example (abbreviated):

```yaml
apiVersion: rbac.authorization.k8s.io/v1
kind: RoleBinding
metadata:
  name: admin
  namespace: autst-feature-16
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: admin
subjects:
  - kind: ServiceAccount
    name: jenkins
    namespace: autst-cd
``` 

When using `oc apply` the role binding named `admin` in rolebinding.yaml (exported from the cloned environment, here autst-dev) wins and replaces the role binding above.
This means jenkins itself no longer has admin provisions and fails fatally.

When using `oc create` however the existing rolebinding created by new-project is not overwritten. However the rolebinding named 'admin' from rolebinding.yaml gets lost.

In projects created with the provisioning app, the rolebinding called `admin` typically have the admin users who where listed during creation and the service account cd/cd-integration in it.
Losing these permissions in the cloned project is typically not a (bigger) problem . 

We tested the changes with our production environment(s).

We aim to target fixing this more fundamentally for ODS 2.0. 

Thanks to @clemensutschig for getting to these findings!
